openwisp_controller/admin_urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .config.views import schema
+
+urlpatterns = [
+    path('netjsonconfig/config/schema.json', schema, name='schema'),
+]
```

Comment:
what about using this file? https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/urls.py

So the solution is also self contained as @pandafy mentioned.